lib/tests/test_conflicts.rs

```diff
@@ -768,8 +768,8 @@ fn test_parse_conflict_crlf_markers() {
 
 #[test]
 fn test_parse_conflict_diff_stripped_whitespace() {
-    // Conflict parsing fails since diff contains empty line without leading space
-    assert_eq!(
+    // Should be able to parse conflict even if diff contains empty line
```

Comment:
I'd add "(without even a leading space, which are sometimes stripped by text editors)" to this comment.